x-pack/test_serverless/api_integration/test_suites/search/config.ts

```diff
@@ -21,5 +22,11 @@ export default createTestConfig({
   kbnServerArgs: [
     // useful for testing (also enabled in MKI QA)
     '--coreApp.allowDynamicConfigOverrides=true',
+    '--xpack.dataUsage.enabled=true',
```

Comment:
```suggestion
    '--xpack.dataUsage.autoops.enabled=true',
```

Reply:
I added `--xpack.dataUsage.autoops.enabled=true`, though left `--xpack.dataUsage.enabled=true` in place so as not to enable the plugin.